bazel/README.md

```diff
@@ -417,5 +417,55 @@ a [JSON Compilation Database](https://clang.llvm.org/docs/JSONCompilationDatabas
 with any tools (e.g. clang-tidy) compatible with the format.
 
 The compilation database could also be used to setup editors with cross reference, code completion.
-For example, you can use [You Complete Me](https://valloric.github.io/YouCompleteMe/) or 
+For example, you can use [You Complete Me](https://valloric.github.io/YouCompleteMe/) or
 [cquery](https://github.com/cquery-project/cquery) with supported editors.
+
+# Advanced caching setup
+
+Below is the advanced caching setup, it will make bazel cache works efficiently when you run
```

Comment:
Setting up an http cache for bazel output helps optimize bazel performance and resource usage when using multiple compilation modes or multiple trees.